cli/src/action/database.rs

```diff
@@ -12,37 +12,170 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use clap::ArgMatches;
+#[cfg(feature = "sqlite")]
```

Comment:
Could all this sqlite code move under its own module?  it would make this module much easier to read